charts/pxc-db/templates/cluster.yaml

```diff
@@ -375,6 +375,13 @@ spec:
     {{- $logcollector := .Values.logcollector }}
     enabled: true
     image: {{ include "pxc-db.logcollector-image" . }}
+    {{- if $logcollector.imagePullPolicy }}
```

Comment:
@agelwarg please add all these options to https://github.com/percona/percona-helm-charts/blob/main/charts/pxc-db/README.md

also please check 
charts/pxc-db/values.yaml
charts/pxc-db/production-values.yaml